dashboard/config/locales/scripts.en.yml

```diff
@@ -10850,7 +10862,9 @@ en:
           lesson_groups: {}
           lessons:
             Aquatic:
-              name: Aquatic
+              name: MINECRAFT Hour of Code
```

Comment:
Is this name change (and the one above) intentional? In particular the "MINECRAFT" in all caps is a bit jarring

Reply:
great question @bethanyaconnor . there is a separate issue going on where, because these are single-lesson units, we do not actually display this lesson name anywhere: https://github.com/code-dot-org/code-dot-org/blob/0037246b1549ddd58484aa537be4f9ef057251c8/dashboard/app/models/lesson.rb#L265-L271

However, this behavior is undesirable, and we do want to start displaying the actual lesson names soon. For now I will update this PR to NOT import these lesson names from the CB lesson plans. A broader plan on updating lesson display names is forthcoming.

Reply:
Sounds good -- thanks for the explanation!

Reply:
this is done, except for ai-ethics and dance-unplugged, which didn't have code studio lessons to begin with, so I let them keep the name that was imported from CB.